en/app.ftl

```diff
@@ -215,7 +215,12 @@ landing-pricing-free-price = Free
 landing-pricing-free-feature-1 = Up to 5 email aliases
 landing-pricing-free-feature-1-2 = Up to 5 email masks
 landing-pricing-free-feature-2 = Browser extension
+landing-pricing-free-feature-free = Free
+landing-pricing-free-description = Try { -brand-name-firefox-relay } aliases and start protecting your email inbox.
```

Comment:
masks?

Also, when are we planning to clean up the old "alias" strings, to avoid confusion?

Reply:
Aaargh! Good eye: ac4cbad

And good point, I'll create a separate PR for that right away, as the content refresh went live the day before yesterday.